linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -832,13 +832,8 @@ private[emitter] final class ClassEmitter(sjsGen: SJSGen) {
         }
       }
 
-      val needIsFunction = !isHijackedClass && (isExpression match {
-        case js.BinaryOp(JSBinaryOp.instanceof, _, _) =>
-          // This is a simple `instanceof`. It will always be inlined at call site.
-          false
-        case _ =>
-          true
-      })
+      val needIsFunction = isHijackedClass || !tree.kind.isClass ||
+          globalKnowledge.isAncestorOfHijackedClass(className)
```

Comment:
That looks like it's going to generate an `$is_` function for hijacked classes, although they're not used. Wasn't this supposed to be
```suggestion
      val needIsFunction = !isHijackedClass && {
        !tree.kind.isClass ||
        globalKnowledge.isAncestorOfHijackedClass(className)
      }
```

Reply:
Yes :-/ Conditionals are hard...